framework/src/play/DefaultConfPropertiesLoader.java

```diff
@@ -11,43 +11,43 @@
 import org.slf4j.LoggerFactory;
 import play.utils.OrderSafeProperties;
 
-public class PropertiesConfLoader implements ConfLoader {
+public class DefaultConfPropertiesLoader implements ConfPropertiesLoader {
```

Comment:
@cies Do we really need to rename `ConfLoader` to `ConfPropertiesLoader` (and `PropertiesConfLoader` to `DefaultConfPropertiesLoader`)?  Isn't it the _invasive change_? ;)

If we rename them back, the overall diff gets much lesser.

Reply:
Fine with me. Reverted the change.